celery_progress/backend.py

```diff
@@ -68,7 +68,7 @@ def get_info(self):
                     'complete': True,
                     'success': success,
                     'progress': _get_completed_progress(),
-                    'result': self.result.get(self.task_id) if success else None,
+                    'result': self.result.get(self.task_id) if success else str(self.result.info),
```

Comment:
is `self.result.info` guaranteed to exist? mostly just wondering if we need to worry about this raising an exception

Reply:
Thanks for considering my feature.

Here's the relevant section in the documentation (https://docs.celeryproject.org/en/stable/reference/celery.result.html):
![image](https://user-images.githubusercontent.com/15802188/80341626-a3aef900-8817-11ea-9b2c-e469a3513611.png)

If there's nothing set, self.result.info returns a _None_ value, so _str(None)_ will just return _'None'_.
If it does throw an exception, the str() function should handle the string conversion also.

Reply:
perfect, thanks.